concourse/scripts/pipelines-bosh-cloudfoundry.sh

```diff
@@ -18,6 +19,9 @@ cf_grafana_version=$("${SCRIPT_DIR}"/val_from_yaml.rb releases.grafana.version "
 generate_vars_file() {
    cat <<EOF
 ---
+pipeline_name: ${pipeline_name}
+makefile_env_target: ${MAKEFILE_ENV_TARGET:-dev}
+self_update_pipeline: ${SELF_UPDATE_PIPELINE:-false}
```

Comment:
Now self update is enabled by default in all environments, unless you specify `SELF_UPDATE_PIPELINE=false`.

Shall I add a note in the README?